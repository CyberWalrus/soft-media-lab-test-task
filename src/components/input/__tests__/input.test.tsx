import React from 'react';
import { shallow } from 'enzyme';
import Input from '../input';

describe('Input component', () => {
  it('should renders correctly', () => {
    const wrapper = shallow(<Input />);
    expect(wrapper.isEmptyRender()).not.toEqual(true);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
