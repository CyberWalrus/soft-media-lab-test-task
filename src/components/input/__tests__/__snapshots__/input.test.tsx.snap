// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should renders correctly 1`] = `"<div class=\\"input\\"><input class=\\"input__item\\"/></div>"`;
